src/modules.js

```diff
@@ -113,6 +113,7 @@ var LibraryManager = {
         'library_fs.js',
         'library_memfs.js',
         'library_tty.js',
+        'library_pipefs.js',
```

Comment:
Oh, I presume the `fs` < `pipefs` order does end up coming from here. In that case, probably worth to add a note here that `pipefs` static init depends on `fs` static init being called before, so the ordering in this list is fixed.